src/framework/ui/text/text.component.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import {
+  Text as TextComponent,
+  TextProps as TextComponentProps,
+} from 'react-native';
+import {
+  StyledComponentProps,
+  StyleType,
+} from '@kitten/theme';
+
+interface TextProps {
+  category?: string;
+}
+
+export type Props = TextProps & StyledComponentProps & TextComponentProps;
+
+export class Text extends React.Component<Props> {
+
+  private getComponentStyle = (source: StyleType): StyleType => {
+    return {
+      color: source.color,
+      fontSize: source.fontSize,
+    };
+  };
+
+  public render(): React.ReactElement<TextProps> {
+    const { style, themedStyle, ...derivedProps } = this.props;
```

Comment:
why not `otherProps`?

Reply:
There are so many variations of naming we can use in spread like `rest` or `other`, I just prefer using `derived` because it is used in each component